src/ctia/domain/id.clj

```diff
@@ -70,13 +73,14 @@
 
 (defn short-id->id
   [type short-id {:keys [hostname path-prefix port protocol]}]
-  (map->CtiaId
-   {:hostname hostname
-    :short-id short-id
-    :path-prefix (not-empty path-prefix)
-    :port port
-    :protocol protocol
-    :type (name type)}))
+  (if (valid-short-id? short-id)
```

Comment:
... I've heard arguments on both sides of the issue. By this point in tie, I figure the debate is so 50-50 that people can just use whichever they like best.